FORMATTING.md

```diff
@@ -41,7 +41,7 @@ impl AccountPatch for EmbeddedAccountPatch {
     }
 }
 ```
-Depending on the case, the readability of the code may decrease, like here (IMHO) since the expanded function body brings none value. 
+Depending on the case, the readability of the code may decrease, since the expanded function body brings none value. 
```

Comment:
>since the expanded function body brings none value
should be "the expanded function brings *no* value. just being pedantic.

Reply:
Wow, why am I paying for writer-grade software to catch exactly this kind of errors? Thanks!